docs/pages/bare/exploring-bare-workflow.md

```diff
@@ -69,4 +69,4 @@ With [Expo Application Services (EAS)](/eas/index.md), you can build and submit
 
 You are now, at a very high level, familiar with the steps you would go through to get started on building an app with the bare workflow. Continue on to [Up and Running](hello-world.md) to get started coding!
 
-Are you feeling intimidated? It might be better for you to start out with the managed workflow if you're new to this. Check out the [managed workflow walkthrough](../introduction/walkthrough.md) for more information.
+Are you feeling intimidated? It might be better for you to start out with the managed workflow if you're new to this. Check out the [First steps](../tutorial/planning/) for more information.
```

Comment:
maybe we should remove this exploring bare workflow page too? it's the same sort of walkthrough

Reply:
I updated it to be sure, but I think the page is already removed actually.

Reply:
it still lives here https://docs.expo.dev/bare/exploring-bare-workflow/